components/automatic-points-table/commons/automatic-points-table.lua

```diff
@@ -0,0 +1,57 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:AutomaticPointsTable
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Class = require('Module:Class')
+local Json = require('Module:Json')
+local Table = require('Module:Table')
+
+local AutomaticPointsTable = Class.new(
+	function(self, frame)
+		self.frame = frame
+		self.args = Arguments.getArgs(frame)
+		self:parseInput(self.args)

```

Comment:
```suggestion
		self.parsedInput = self:parseInput(self.args)

```

Reply:
@JaspervanRiet I need to keep track of the parsed input in order to use it later, so is it fine to add it like so, to be able to access it in the main function?

Reply:
Probably best to set `self.tournaments`, etc in `parseInput

Reply:
No yea actually, this seems good.